Magento2/Sniffs/Annotation/MethodArgumentsSniff.php

```diff
@@ -70,7 +70,23 @@ private function isTokenBeforeClosingCommentTagValid(string $type): bool
     private function validateCommentBlockExists(File $phpcsFile, int $previousCommentClosePtr, int $stackPtr): bool
     {
         $tokens = $phpcsFile->getTokens();
+        $attributeFlag = false;
         for ($tempPtr = $previousCommentClosePtr + 1; $tempPtr < $stackPtr; $tempPtr++) {
+            $tokenCode = $tokens[$tempPtr]['code'];
+
+            // Ignore attributes e.g. #[\ReturnTypeWillChange]
```

Comment:
What do you think about just adding T_ATTRIBUTE (and other attribute related tokens) to `validTokensBeforeClosingCommentTag` property? (would be good to actually change this property to const)

I believe the idea of the `validTokensBeforeClosingCommentTag` token codes is actually to list tokens allowed between comment and function - sounds like exactly what we need.

Reply:
Hi @sivaschenko , as we don't have a single token for the attribute, in that case we need to add T_ATTRIBUTE, T_ATTRIBUTE_END, T_NS_SEPARATOR, T_STRING  and other tokens that possibly can be in other attributes in the future,
e.g. something like
```
#[ListensTo(ProductCreated::class)]
#[\Attribute(\Attribute::TARGET_METHOD | \Attribute::TARGET_PROPERTY)]
```

So I think it's better just to ignore everything between #[ and ]